<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoosePageSaveFormatDlg</class>
 <widget class="QDialog" name="ChoosePageSaveFormatDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Save Format</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>How would you like to save the page?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_singleHTMLPage">
     <property name="text">
      <string>As a &amp;Single HTML page</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_asDirectory">
     <property name="text">
      <string>As &amp;directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_asMHTMLPage">
     <property name="text">
      <string>As single MHTML file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_info">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChoosePageSaveFormatDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChoosePageSaveFormatDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
